spec/services/login_adoption_service_spec.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe LoginAdoptionService do
+  let(:service) { LoginAdoptionService.new(user) }
+
+  context 'organic modal experiment' do
+    describe 'a dslogon user' do
+      let(:user) { create(:user, :dslogon) }
+
+      it 'organic modal flag is off' do
+        Flipper.disable(:organic_dsl_conversion_experiment)
+        expect(service.perform).to include(organic_modal: false)
```

Comment:
Completely a nitpick (feel free to ignore if you'd like), but for unit specs I tend to prefer exact expectations if at all possible. In this case, you know that `service.perform` is exactly `{ organic_modal: true }`, so IMO you may as well expect exactly that hash.

Reply:
I use include because in the past I found testing for the exact return can result in a lot of needlessly breaking specs as it evolves. If you feel strongly in the future I can change it, but for now I'd like to leave as is.